eth/contracts/AbstractMost.sol

```diff
@@ -381,6 +397,12 @@ abstract contract AbstractMost is
         wrappedAzeroAddress = _wrappedAzeroAddress;
     }
 
+    function setTransferLimit(
+        ITransferLimit _transferLimit
+    ) external onlyOwner {
```

Comment:
Should it have `whenPaused` here?

Reply:
I don't think it's needed, adding/removing/changing won't invalid any requests already in the queue.